src/views/Dashboard/Lab/Services/index.vue

```diff
@@ -189,7 +192,7 @@ export default {
       if(val && val != ""){
          return val
       }
-      return "https://ipfs.io/ipfs/QmaGr6N6vdcS13xBUT4hK8mr7uxCJc7k65Hp9tyTkvxfEr"
+      return "https://ipfs.io/ipfs/QmZveuvcQAbkcbcop28iqWrnCMoxN2n5SGzA3Hwr1cJ61i/debio-logo.png"
```

Comment:
The DeBio logo shouldn't be hardcoded like this. Wouldn't it be better to turn it into an Icon? @luthfiazhari.

Please ask @bukandicki for help. @bukandicki, please help @luthfiazhari with this.

Reply:
Example: https://github.com/debionetwork/debio-customer-ui/pull/309/commits/1c248c86dc174be12e1d4b3a1e4a55791a4ca96a